[2025-01-05T09:53:34.993+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.call_dm_fill_data_marts_f scheduled__2025-01-04T00:00:00+00:00 [queued]>
[2025-01-05T09:53:35.004+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.call_dm_fill_data_marts_f scheduled__2025-01-04T00:00:00+00:00 [queued]>
[2025-01-05T09:53:35.004+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-05T09:53:35.157+0000] {taskinstance.py:2191} INFO - Executing <Task(SQLExecuteQueryOperator): call_dm_fill_data_marts_f> on 2025-01-04 00:00:00+00:00
[2025-01-05T09:53:35.164+0000] {standard_task_runner.py:60} INFO - Started process 129 to run task
[2025-01-05T09:53:35.167+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'call_dm_fill_data_marts_f', 'scheduled__2025-01-04T00:00:00+00:00', '--job-id', '359', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpb8o_ub3o']
[2025-01-05T09:53:35.169+0000] {standard_task_runner.py:88} INFO - Job 359: Subtask call_dm_fill_data_marts_f
[2025-01-05T09:53:35.210+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.call_dm_fill_data_marts_f scheduled__2025-01-04T00:00:00+00:00 [running]> on host 403e7aadd7ee
[2025-01-05T09:53:35.272+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='agushchin' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='call_dm_fill_data_marts_f' AIRFLOW_CTX_EXECUTION_DATE='2025-01-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-04T00:00:00+00:00'
[2025-01-05T09:53:35.273+0000] {sql.py:276} INFO - Executing: CALL dm.dm_fill_data_marts_f('scheduled__2025-01-04T00:00:00+00:00')
[2025-01-05T09:53:35.280+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-05T09:53:35.286+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-05T09:53:35.295+0000] {sql.py:450} INFO - Running statement: CALL dm.dm_fill_data_marts_f('scheduled__2025-01-04T00:00:00+00:00'), parameters: None
[2025-01-05T09:53:35.547+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=call_dm_fill_data_marts_f, execution_date=20250104T000000, start_date=20250105T095334, end_date=20250105T095335
[2025-01-05T09:53:35.581+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-05T09:53:35.601+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
